BuildModule.cmake

```diff
@@ -1,4 +1,16 @@
-include (../../flags.cmake)
+include (${build_environment}/flags.cmake)
+
+set (sources ${sources}
```

Comment:
In de nieuwste build_environment hoeft je niet meer de main.cpp, wrap-hwlib.cpp en lubc-stub.cpp te includen, alleen nog maar je eigen sources.